react-webapp/src/serviceWorker.js

```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
```

Comment:
Does this file come from us? Does it need license/copyright info?

Reply:
That comes with https://github.com/facebook/create-react-app which I used to create this app. I didn't delete it yet as I was trying to make as few changes as possible (and it's possible you'd want to re-enable it if you wanted to build a react app. But I guess it's not needed and would be cleaner without it so I'm happy to delete it.